slotfills/sections/richtext-field.tsx

```diff
@@ -1,17 +1,30 @@
 /* global tinyMCEPreInit */
 import React, { useState } from 'react';
-import PropTypes from 'prop-types';
 import { PanelRow } from '@wordpress/components';
 import { Editor } from '@tinymce/tinymce-react';
+import Field from '@/interfaces/field';
 
-const RichtextField = ({
+declare global {
+  const tinyMCEPreInit: {
+    baseURL: string,
+  };
+}
+
+type Props = {
+  field: Field,
+  valueHook: Function,
```

Comment:
You can go a step further here and define a function signature. For example, if it's a function that accepts no parameters and returns nothing, it would be `() => void`. You could add parameters, return types, etc. That'll help with additional code checks and type hinting. See, for example, https://github.com/alleyinteractive/wp-starter-plugin/blob/feature/typescript/components/image-picker/index.tsx#L8-L10